multimodal/src/autogluon/multimodal/utils/environment.py

```diff
@@ -8,12 +8,8 @@
 import torch
 from torch import nn
 
-try:
-    from mmcv.parallel import DataContainer
-except:
-    pass
-
 from ..constants import AUTOMM, OBJECT_DETECTION, OCR
+from .mmcv import DataContainer
```

Comment:
How about still keeping the try-except, but only replace `pass` with `from .mmcv import DataContainer`?